js/bybit.js

```diff
@@ -4012,15 +4011,68 @@ module.exports = class bybit extends Exchange {
             currency = this.currency (code);
             request['coin'] = currency['id'];
         }
+        const depositMethodCutoff = 1626393600000;
+        let method = 'privateGetAssetV1PrivateDepositRecordQuery';
+        const until = this.safeNumber (params, 'until');
         if (since !== undefined) {
-            request['start_date'] = this.yyyymmdd (since);
+            request['wallet_fund_type'] = 'Deposit'; // Deposit, Withdraw, RealisedPNL, Commission, Refund, Prize, ExchangeOrderWithdraw, ExchangeOrderDeposit
+            if (since <= depositMethodCutoff) {
+                method = 'privateGetV2PrivateWalletFundRecords';
+                request['start_date'] = this.yyyymmdd (since);
+            } else {
+                const startTime = since / 1000;
+                request['start_time'] = startTime;
+                const thirtyDays = 2592000;
+                if (until === undefined) {
+                    request['end_time'] = startTime + thirtyDays;
```

Comment:
Can't use + for arithmetic addition here.

```suggestion
                    request['end_time'] = this.sum (startTime, thirtyDays);
```